plugins/analysis/webinterface/httpserver/static/js/main.js

```diff
@@ -580,13 +488,6 @@ class Application {
             ((window.location.protocol === "https:") ? "wss://" : "ws://") + this.url
         );
     
-        this.socket.onopen = () => {
-            this.setStatusMessage("WebSocket opened. Loading ... ");
-            setInterval(() => {
-                this.socket.send("_");
```

Comment:
This served as a sort of heartbeat/ping pong message. I'm not entirely sure if we can remove that without leading to disconnects. See here: https://stackoverflow.com/questions/10585355/sending-websocket-ping-pong-frame-from-browser

Maybe the Go websocket module brings support for it so it could be implemented from server side. Like the node.js library: https://github.com/websockets/ws/issues/977

Reply:
It worked without this, but to be on the safe side, I added it back.